File: app/src/main/java/org/techtown/find_gas_station/Retrofit/oilList/RESULT.java
package org.techtown.find_gas_station.Retrofit.oilList;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.techtown.find_gas_station.Retrofit.oilList.OIL;

public class RESULT
{
    @SerializedName("OIL")
    @Expose
    private org.techtown.find_gas_station.Retrofit.oilList.OIL[] OIL;

    public OIL[] getOIL ()
    {
        return OIL;
    }

    public void setOIL (OIL[] OIL)
    {
        this.OIL = OIL;
    }

    @Override
    public String toString()
    {
        return "ClassPojo [OIL = "+OIL+"]";
    }
}

File: app/src/main/java/org/techtown/find_gas_station/domain/stationavgprice/model/Result.kt
package org.techtown.find_gas_station.domain.stationavgprice.model

import com.google.gson.annotations.SerializedName

data class Result(

    @SerializedName("OIL")
    val oilAveragePriceInfo : List<OilAveragePriceInfo>
)
